modules/distribution/src/policies/pages/index.hbs

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
```

Comment:
license is missing.

Reply:
Fixed with https://github.com/wso2/product-sp/pull/1014